cmd/sgx_epc_extendedresource/main.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"os"
+
+	"github.com/klauspost/cpuid/v2"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/klog/v2"
+)
+
+const resourceName = "sgx.intel.com~1epc"
+
+type patchExtendedResource struct {
+	Op    string `json:"op"`
+	Path  string `json:"path"`
+	Value uint64 `json:"value"`
+}
+
+func main() {
+	var affirm bool
+	flag.BoolVar(&affirm, "affirm", false, "return error if EPC is not available")
+	flag.Parse()
```

Comment:
Could this functionality also be part of  `sgx_epchook` with an extra `-register` opt-in boolean to register to the API server directly instead of just print?

Reply:
Good catch, thanks! Will update the PR